CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # MicroProfile Language Server Changelog
 
+## [0.7.0](https://github.com/eclipse/lsp4mp/milestone/7?closed=1) (March 28, 2023)
+
+### Enhancements
+ * Validation for GraphQL `@Query` and `@Mutation` methods with `void` return type. See [#348](https://github.com/eclipse/lsp4mp/issues/348), [#359](https://github.com/eclipse/lsp4mp/issues/359).
+ * Navigate to REST endpoints using WorkspaceSymbols. See [#87](https://github.com/eclipse/lsp4mp/issues/87).
```

Comment:
I would maybe call it "workspace symbols". Doing that means if we refer to this feature on the extension side, it's a mere copy/paste, as opposed to having to rename it bcause "WorkspaceSymbols" is not really a term we should exposing to users.